addons/binding/org.openhab.binding.solaredge/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,36 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: SolarEdge Binding
+Bundle-SymbolicName: org.openhab.binding.solaredge;singleton:=true
+Bundle-Vendor: openHAB
+Bundle-Version: 2.3.0.qualifier
+Bundle-RequiredExecutionEnvironment: JavaSE-1.8
+Bundle-ClassPath: .
+Import-Package: com.google.common.collect,
```

Comment:
We are trying to get rid off Guave - as you hardly use it in your code, could you refactor those places to work without Guava?

Reply:
Hmm there is only one reference to it in my constants:
public static final Set<ThingTypeUID> SUPPORTED_THING_TYPES_UIDS = ImmutableSet.of(THING_TYPE_GENERIC);

I found this in other bindings as well. I can switch to a mutable collections but it is of course better to have a constant that is really immutable.

Reply:
As I currently only have one item in my set I can use this alternative:
    public static final Set<ThingTypeUID> SUPPORTED_THING_TYPES_UIDS = Collections.singleton(THING_TYPE_GENERIC); 

If I add more elements I will have to use Collections.unmodifiableSet instead which needs to be constructed from a modifiable Set which then will be a private property.